test/integration/configuration.test.js

```diff
@@ -60,15 +60,29 @@ describe(Support.getTestDialectTeaser('Configuration'), () => {
       }
 
       const seq = new Sequelize(config[dialect].database, config[dialect].username, 'fakepass123', { logging: false, host: config[dialect].host, port: 1, dialect });
-      if (dialect === 'sqlite') {
+      switch (dialect) {
+        case 'sqlite': {
         // SQLite doesn't require authentication and `select 1 as hello` is a valid query, so this should be fulfilled not rejected for it.
```

Comment:
I think we should inline this a little bit

Reply:
I actually didn't mean to convert to switch, it's a rule we disabled in our local config, that I disabled in the base config, but it I auto-fixed before updating the base config :/

I'll revert the switch changes

Reply:
Just adding two extra spaces at the start so the indentation of the exepct is the same as the comment